nginx/modsecurity/config.in

```diff
@@ -24,7 +24,8 @@ CORE_LIBS="$CORE_LIBS \
     @YAJL_LIBS@ \
     @SSDEEP_LDFLAGS@ \
     -lstdc++ \
-    -lprotobuf"
+    /usr/lib/libprotobuf.a"
```

Comment:
I think it would be better to put the location path (/usr/lib/) into common paths where libraries are being searched at and only specify the static library name here.
Linking paths are specified using `-L` compiler flag and libraries names use `-l`.

Reply:
Found a way that avoids the hardcoded paths.
    -Wl,-Bstatic
    -lprotobuf
    -Wl,-Bdynamic